client/apps/edit2/components/display/test/social.test.js

```diff
@@ -6,9 +6,9 @@ import { Fixtures } from '@artsy/reaction/dist/Components/Publishing'
 import { Provider } from 'react-redux'
 import { CharacterLimit } from 'client/components/character_limit'
 import { DisplaySocial } from '../components/social'
-import ImageUpload from 'client/apps/edit2/components/admin/components/image_upload.coffee'
+import ImageUpload from '../../admin/components/image_upload.coffee'
 
-describe('DisplaySocial', () => {
+xdescribe('DisplaySocial', () => {
```

Comment:
I see we're pending quite a few tests but I presume these will work when we swap edit2?

Reply:
Yeah they are passing locally, so i think will be easier to debug when everything is switched over.